ansible/host_vars/datacollector.infra.ooni.io/vars.yml

```diff
@@ -10,10 +10,14 @@ airflow_fernet_key: "{{ vault_airflow_fernet_key }}"
 airflow_redis_password: "{{ vault_airflow_redis_password }}"
 airflow_postgres_password: "{{ vault_airflow_postgres_password }}"
 
-# tokens to read https://s3.amazonaws.com/ooni-* buckets
+# tokens to read https://s3.amazonaws.com/ooni-* buckets from Open Observatory CEO personal S3 account
 s3read_access_key_id: "{{ vault_s3read_access_key_id }}"
 s3read_secret_access_key: "{{ vault_s3read_secret_access_key }}"
```

Comment:
@hellais is that reference good enough? I used the one that was used in the mail thread with Jim Ying :-)